tracker/index.js

```diff
@@ -1,29 +1,19 @@
-const express = require('express');
+const express = require("express");
 const app = express();
-const { ExpressPeerServer } = require('peer');
+const http = require("http").createServer(app);
+const io = require("socket.io")(http);
 
-app.get('/', (req, res, next) => { 
-    console.log('GET /')
-    res.send('Tracker Online!'); 
-});
-
-const server = app.listen(9000);
+let session_list = ['test1', 'test2', 'test3'];
 
-const options = {
-    debug: true,
-    path: '/peerjs'
-}
+io.on("connection", function(socket) {
 
-const peerserver = ExpressPeerServer(server, options);
```

Comment:
Although it doesn't do much, if we are still using `peerjs` for P2P communication, we might want to keep the peerjs server running here as well. Otherwise we'll need to depend on the public `peerjs` server.

Reply:
Okie!